site/gatsby-site/src/components/checklists/CheckListForm.js

```diff
@@ -0,0 +1,437 @@
+              textClasses="text-2xl"
+              iconClasses="text-lg vertical-align"
+            />
+          </h1>
+          <HeaderControls>
+            <SavingIndicator {...{ saveStatus }} />
+            <Button color="light" onClick={() => alert('Coming soon')}>
+              <Trans>Subscribe</Trans>
+            </Button>
+            <DeleteButton type="button" onClick={() => confirmDeleteChecklist(values.id)}>
+              Delete
+            </DeleteButton>
+            <ExportDropdown checklist={values} />
+          </HeaderControls>
+        </HeaderRow>
+      </Header>
+      <Info>
+        This feature is in development.
+        Data entered will not be retained.
+      </Info>
+      {/*<Info className="my-4">
+        <Trans>
+          Describe the system under investigation. Apply machine-readable tags to surface risks
+          associate with likewise-tagged incidents in the database. These are the basis of a risk
+          checklist.
+        </Trans>
+      </Info>*/}
+      <section>
+        <Label for="about-system">About System</Label>
+        <Textarea
+          placeholder={t(`Human-readable notes on the system under investigation`)}
+          value={values.about}
+          row={4}
+          onChange={(event) => {
+            setFieldValue('about', event.target.value);
+          }}
+        />
+      </section>
+      <section>
+        <SideBySide className="my-4">
+          <GoalsTagInput {...{ values, tags, setFieldValue }} />
+          <MethodsTagInput {...{ values, tags, setFieldValue }} />
+        </SideBySide>
+        <OtherTagInput {...{ values, tags, setFieldValue }} />
+      </section>
+
+      {/* TODO: Remove this */}
+      <section>
+        <details>
+          <summary>allPrecedents</summary>
+          <pre>{JSON.stringify(allPrecedents, null, 2)}</pre>
+        </details>
+      </section>
+
+      <section>
+        <header className="flex mt-6">
+          <h2>Risks</h2>
+          <div className="flex gap-2 ml-auto">
+            <Button
+              color="light"
+              onClick={() =>
+                setFieldValue(
+                  'risks',
+                  values.risks.map((r) => ({ ...r, startClosed: false }))
+                )
+              }
+            >
+              Expand all
+            </Button>
+            <Button
+              color="light"
+              onClick={() =>
+                setFieldValue(
+                  'risks',
+                  values.risks.map((r) => ({ ...r, startClosed: true }))
+                )
+              }
+            >
+              Collapse all
+            </Button>
+            <Button
+              onClick={() => {
+                setFieldValue('risks', [emptyRisk()].concat(values.risks || []));
+              }}
+            >
+              Add Risk
+            </Button>
+          </div>
+        </header>
+        {/*<Info>
+          <Trans>
```

Comment:
Noting the commented code here. Might be acceptable to leave during prototyping but just FYI.

Reply:
I'm unsure whether we need help text so for now I'd like to keep it there.